### PR TITLE
Sfcc/ex 8

### PR DESCRIPTION
**Pull request by Khalevskyi**

### CODE REVIEW

File: cartridges/int_extend_sfra/cartridge/scripts/jobHelpers.js [hunk 1]
```diff
@@ -32,8 +32,54 @@ function getContractLoggerModel (order) {
...
+
+            if (status === refundStatus.ERROR) {
+                error = true;
+            } else if (status === refundStatus.REJECT) {
+                reject = true;
+            }
+        }
+    }
+
+    if (error && reject) {
```
Comment (pkovalyov) [inline cartridges/int_extend_sfra/cartridge/scripts/jobHelpers.js:68]: please do not duplicate the code, use helpers

File: cartridges/int_extend/cartridge/scripts/jobHelpers.js [hunk 2]
```diff
@@ -32,8 +32,54 @@ function getContractLoggerModel (order) {
...
+
+            if (status === refundStatus.ERROR) {
+                error = true;
+            } else if (status === refundStatus.REJECT) {
+                reject = true;
+            }
+        }
+    }
+
+    if (error && reject) {
```
[the remark this answers is not in the document]
Comment (Khalevskyi) [reply cartridges/int_extend/cartridge/scripts/jobHelpers.js:68]: An order may contain a couple of contracts and they may have different statuses. REJECT if the sum of the refund is equal to 0 and an ERROR if some error happened during the service call or connection error occurred.

File: cartridges/int_extend_sfra/cartridge/scripts/extend.js [hunk 3]
```diff
@@ -98,9 +98,41 @@ function createProduct(paramData) {
...
+ *
+ * @param {Object} paramData - query params
+ * @returns {Object} - response object
+ */
+ function createRefund(paramData) {
+    var serviceResponse = null;
+
+    serviceResponse = webService.createRefundRequest(paramData);
+
+    if (!serviceResponse.ok) {
```
[the remark this answers is not in the document]
Comment (Khalevskyi) [reply cartridges/int_extend_sfra/cartridge/scripts/extend.js:112]: This is not a duplication, but another cartridge with the same code.

File: cartridges/int_extend_sfra/cartridge/scripts/jobHelpers.js (hunk 1 above)
Comment (Khalevskyi) [reply cartridges/int_extend_sfra/cartridge/scripts/jobHelpers.js:68]: This is not a duplication, but another cartridge with the same code.

**Pull request merged**